### ISSUE TITLE
No python 3.8 wheels for tbb4py and missing "python_requires" in package metadata 

### ISSUE THREAD

**Issue by ashutoshvarma** [oneapi-src/oneTBB#325]
When I was trying to test `tbb` in python 3.8.6, I kept getting "module not found" despite installing `tbb4py`  using pip.

```console
❯ python --version
Python 3.8.6

❯ pip install tbb4py
Collecting tbb4py
  Using cached tbb4py-2021.1.1-py37-none-manylinux1_x86_64.whl (427 kB)
Collecting tbb==2021.1.1
  Using cached tbb-2021.1.1-py2.py3-none-manylinux1_x86_64.whl (3.6 MB)
Installing collected packages: tbb, tbb4py
Successfully installed tbb-2021.1.1 tbb4py-2021.1.1

❯ python -m tbb -h
/home/oreki_clr/.pyenv/versions/tbb_test/bin/python: No module named tbb
```

After looking at [pypi ](https://pypi.org/project/tbb4py/#files) I found that there is no py3.8 wheel and sdist. (It would be helpful if you can publish sdist along with wheels.)

Ideally installing `tbb4py` in py3.8 should raise an error but it instead silently install py3.7 wheels in py3.8.  (probably due to missing sdist or `python_requires` in setup.py ?)

**System Info**
**OS** - ClearLinux
**Python** - 3.8.6
**setuptools** - 51.1.0
**pip** -  20.3.3
**wheel** - 0.36.2

**Comment by anton-potapov** [comment]
@AlexVeprev  could you please take a look?

**Comment by AlexVeprev** [comment]
Yes, there is a gap with missing requirement, we're working on resolution with distribution team. @anton-potapov, thank you for highlighting!

**Comment by anton-potapov** [comment]
@AlexVeprev ping

**Comment by AlexVeprev** [comment]
It's still WIP

**Comment by isaevil** [comment]
@ashutoshvarma As I can see, tbb4py for py3.8 is now listed https://pypi.org/project/tbb4py/#files. Do you still experience this issue?
